codecs/opus/src/main/jni/OpusJni.sln

```diff
@@ -0,0 +1,28 @@
+﻿
```

Comment:
Should this file be part of .gitignore?

Reply:
It is needed for Windows native library build. It can be removed later if Windows is not going to be supported.